website/content/en/getting-started/usage/_index.md

```diff
@@ -82,7 +82,7 @@ To provide identity to access key vault, refer to the following [section](#provi
   | objectFormat           | no       | [__*available for version > 0.0.7*__] the format of the Azure Key Vault object, supported types are pem and pfx. `objectFormat: pfx` is only supported with `objectType: secret` and PKCS12 or ECC certificates        | "pem"         |
   | objectEncoding         | no       | [__*available for version > 0.0.8*__] the encoding of the Azure Key Vault secret object, supported types are `utf-8`, `hex` and `base64`. This option is supported only with `objectType: secret`                      | "utf-8"       |
   | filePermission         | no       | [__*available for version > v1.1.0*__] permission for secret file being mounted into the pod                      | "0644"       |
-  | tenantId               | yes      | tenant ID containing key vault instance                                                                                                                                                                                | ""            |
+  | tenantId               | yes      | tenant ID containing the Key Vault instance. Should be set to `"adfs"` for [custom clouds](../../configurations/custom-environments) using the AD FS identity provider system                                                                       | ""            |
```

Comment:
Is it always `adfs` for custom cloud or only for Azure Stack Hub?

Reply:
You are right, I am not aware of any other custom clouds using `AD FS`, but that's not enough to justify the statement, will clarify